### PR TITLE
refreshResizeFrame

### PR DESCRIPTION
**Pull request by Kuznecoff**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,6 @@
 .idea/
 node_modules/
 .vscode/
-dist/
```
Comment (myshov) [inline .gitignore]: Why do we need dist inside the repo?

**Review by myshov — CHANGES_REQUESTED** [review]
Issue with dist inside the repo

**Review by Farfurix — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Kuznecoff) [reply .gitignore]: Its need temporary, i remove it before merge

**Review by myshov — APPROVED** [review]
Thanks!

**Pull request merged**